packages/annotorious/src/model/w3c/fragment/FragmentSelector.ts

```diff
@@ -1,16 +1,6 @@
-import type { W3CSelector } from '@annotorious/core';
-import { ShapeType } from '../../core';
 import type { Rectangle, RectangleGeometry } from '../../core';
-
-export interface FragmentSelector extends W3CSelector {
-
-  type: 'FragmentSelector';
-
-  conformsTo: 'http://www.w3.org/TR/media-frags/',
-
-  value: string;
-  
-}
```

Comment:
It's debatable whether `FragmentSelector` and `SVGSelector` should have been moved to the `W3CImageAnnotation.ts`. But in my view it's a bit easier to observe the whole type in one place